pkg/apis/minio.min.io/v1/helper.go

```diff
@@ -278,11 +278,6 @@ func (t *Tenant) AllMinIOHosts() []string {
 
 // MinIOCIServiceHost returns ClusterIP service Host for current Tenant
 func (t *Tenant) MinIOCIServiceHost() string {
-	if t.Spec.Zones[0].Servers == 1 {
-		msg := "Please set the server count > 1"
```

Comment:
What is the original logic behind this check?

Reply:
It was probably being used to build the arguments but now we have a separate function to build arguments